test/refunder-test.ts

```diff
@@ -0,0 +1,295 @@
+
+		});
+	
+		it("Not Owner should get max gas price", async function() {
+			const gasPrice = await refunder.connect(addr1).maxGasPrice();
+			
+			expect('0').to.be.eq(gasPrice.toString());
+		});
+
+		it("Not Owner should NOT set max gas price", async function() {
+			await expect(refunder.connect(addr1).setMaxGasPrice(191)).to.be.revertedWith(NOT_AN_OWNER);
+		});
+	});
+
+	describe('Whitelist Refundable', () => {
+
+		it("Owner should be able to check for refundable", async function() {
+		
+			const randomAddress = greeter.address;
+			const randomFuncId = ethers.utils.id('setGreeting(string)');
+			const randomFuncIdAsBytes = ethers.utils.arrayify(randomFuncId.substr(0, 10))
+
+			// address, bytes4
+			const res = await refunder.refundables(randomAddress, randomFuncIdAsBytes); 
+			
+			expect(res).to.be.eq(false);
+		});
+
+		it("Owner should be able to add refundable", async function() {
+		
+			const randomAddress = greeter.address;
+			const randomFuncId = ethers.utils.id('setGreeting(string)');
+			const randomFuncIdAsBytes = ethers.utils.arrayify(randomFuncId.substr(0, 10))
+
+			// address, bytes4
+			const resBefore = await refunder.refundables(randomAddress, randomFuncIdAsBytes); 
+			expect(resBefore).to.be.eq(false);
+
+			const res = await refunder.whitelistRefundable(randomAddress, randomFuncIdAsBytes, true);
+			await res.wait();
+
+			const resAfter = await refunder.refundables(randomAddress, randomFuncIdAsBytes); 
+			expect(resAfter).to.be.eq(true);
+		});
+
+		it("Owner should be able to edit refundable", async function() {
+		
+			const randomAddress = greeter.address;
+			const randomFuncId = ethers.utils.id('setGreeting(string)');
+			const randomFuncIdAsBytes = ethers.utils.arrayify(randomFuncId.substr(0, 10))
+
+			// address, bytes4
+			const resBefore = await refunder.refundables(randomAddress, randomFuncIdAsBytes); 
+			expect(resBefore).to.be.eq(false);
+
+			let res = await refunder.whitelistRefundable(randomAddress, randomFuncIdAsBytes, true);
+			await res.wait();
+
+			const resAfter = await refunder.refundables(randomAddress, randomFuncIdAsBytes); 
+			expect(resAfter).to.be.eq(true);
+
+			res = await refunder.whitelistRefundable(randomAddress, randomFuncIdAsBytes, false);
+			await res.wait();
+
+			const resAfterEdit = await refunder.refundables(randomAddress, randomFuncIdAsBytes); 
+			expect(resAfterEdit).to.be.eq(false);
+		});
+	
+		it("Not Owner should be able to check for refundable", async function() {
+			const randomAddress = greeter.address;
+			const randomFuncId = ethers.utils.id('setGreeting(string)');
+			const randomFuncIdAsBytes = ethers.utils.arrayify(randomFuncId.substr(0, 10))
+
+			// address, bytes4
+			const res = await refunder.connect(addr1).refundables(randomAddress, randomFuncIdAsBytes); 
+			
+			expect(res).to.be.eq(false);
+		});
+
+		it("Not Owner should NOT be able to add, edit refundable", async function() {
```

Comment:
Lets rename this to `Not owner should NOT be able to add refundable` and add another test for: `Not owner should NOT be able to edit refundable`